Simplenote/NoteTableCellView.swift

```diff
@@ -0,0 +1,161 @@
+import Foundation
+import AppKit
+
+
+// MARK: - TagTableCellView
+//
+@objcMembers
+class NoteTableCellView: NSTableCellView {
+
+    /// TextField: Title
+    ///
+    @IBOutlet private var titleTextField: NSTextField!
+
+    /// TextField: Body
+    ///
+    @IBOutlet private var bodyTextField: NSTextField!
+
+    /// LeftImage: Pinned Indicator
+    ///
+    @IBOutlet private var leftImageView: NSImageView!
```

Comment:
Nitpicks, but if this is always used for pinned and the right is always used for shared, should we just name them `pinnedImageView` and `sharedImageView`?

Reply:
Done!! MUCH better, thank you!!